### PR TITLE
Added support to launch test jobs without re-uploading files

### PR DESCRIPTION
**Pull request by iamshital**

### CODE REVIEW

File: JenkinsPipelines/Scripts/ReceivePartnerFiles.ps1 [hunk 1]
```diff
@@ -111,22 +101,14 @@ if ($env:Kernel -eq "custom")
         }
         else
         {
+            <#
             LogMsg "Downloading $($env:customKernelURL)"
             $out = Start-BitsTransfer  -Source "$env:customKernelURL"
             if ($?)
             {
-                if ($env:customKernelURL.EndsWith(".deb"))
```
Comment (nicepat) [inline JenkinsPipelines/Scripts/ReceivePartnerFiles.ps1]: Is it ok with any file extension?

File: JenkinsPipelines/Scripts/ReceivePartnerFiles.ps1 [hunk 2]
```diff
@@ -160,8 +143,10 @@ if ($env:CustomVHD)
     {
         
 
-        LogMsg "Copying '$env:CustomVHD' --> $PartnerUsernameShareDirectory\$BuildNumber-$env:CustomVHD"
-        Move-Item CustomVHD $PartnerUsernameShareDirectory\$BuildNumber-$env:CustomVHD -Force
+        #LogMsg "Copying '$env:CustomVHD' --> $PartnerUsernameShareDirectory\$BuildNumber-$env:CustomVHD"
```
Comment (nicepat) [inline JenkinsPipelines/Scripts/ReceivePartnerFiles.ps1]: remove comment

File: XML/TestCases/SmokeTests.xml [hunk 3]
```diff
@@ -12,9 +12,9 @@
 	</test>
 	<test>
 		<TestName>VERIFY-HOSTNAME</TestName>
-		<testScript>BVT-VERIFY-HOSTNAME.py</testScript>
-		<PowershellScript>BVT-VERIFY-HOSTNAME.ps1</PowershellScript>
-		<files>.\Testscripts\Linux\nslookup.py,.\Testscripts\Linux\BVT-VERIFY-HOSTNAME.py,.\Testscripts\Linux\azuremodules.py</files>
+		<testScript>WALA-VERIFY-HOSTNAME.py</testScript>
```
Comment (nicepat) [inline XML/TestCases/SmokeTests.xml]: 'tag' should change to 'wala'?

Comment (iamshital) [reply XML/TestCases/SmokeTests.xml]: added tag wala

**Pull request merged**